slack-base/src/test/resources/conversation_kick_response.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ok": true,
+  "response_metadata": {
+    "next_cursor": ""
```

Comment:
QQ, why do we have this param in place? Is it to test that the model still works when the response contains a new unexpected field?

Reply:
Actually this field is in SlackResponse form which every response classes extend, but in ConversationKickResponse we don't need that.